Tabloid/client/src/modules/TagView.js

```diff
@@ -0,0 +1,32 @@
+import { getAllTags } from '../components/TagManagment';
```

Comment:
This file contains a component. This file should be in the components directory.

Reply:
@heymonicakay  Hey I changed the tagView and moved it to the components directory